src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -58,12 +59,33 @@ public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext
             DocumentCollectionInfo documentCollectionLocation;
             DocumentCollectionInfo leaseCollectionLocation;
             ChangeFeedProcessorOptions processorOptions = BuildProcessorOptions(attribute);
+            
             processorOptions.StartFromBeginning = attribute.StartFromBeginning;
+
             if (attribute.MaxItemsPerInvocation > 0)
             {
                 processorOptions.MaxItemCount = attribute.MaxItemsPerInvocation;
             }
 
+            if (!string.IsNullOrEmpty(attribute.StartFromTime))
+            {
+                if (attribute.StartFromBeginning)
+                {
+                    throw new InvalidOperationException("Only one of StartFromBeginning or StartFromTime can be used");
+                }
+
+                if (!DateTime.TryParse(attribute.StartFromTime, out DateTime startFromTime))
+                {
+                    throw new InvalidOperationException("Specified StartFromTime parameter is not in appropriate DateTime format. Please refer: https://docs.microsoft.com/dotnet/api/system.datetime.parse?view=net-5.0#parsing-and-cultural-conventions");
+                }
```

Comment:
This link captures all the possible combinations that the API can handle. Its a long doc to replicate?

Reply:
Whoops -- I accidentally deleted your comment when I was trying to delete a draft comment of mine!

Reply:
*Here's my comment in response to yours...*

Yes, but that link may break at some point in the future (it has `net-5.0` in the URL, for example). Also, this may be used by languages other than C# in Functions, which means linking to .NET `DateTime.Parse()` documentation can be confusing. 

I'd prefer something like this, which gives an explicit example right in the error message and doesn't mention .NET.:

`The specified StartFromTime parameter is not in the correct format. Please use the ISO 8601 format with the UTC designator. For example: "2021-02-16T14:19:29Z".`

As I mentioned in the doc comments for the `StartFromTime` property: Yes, other formats will work, but this one is guaranteed to work and reduces the need to lookup a separate documentation page and look through all of the options.